modules/oadp-operator-issues.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-operator-issues_{context}"]
+= OADP Operator issues
+
+You might encounter issues caused by problems with the {oadp-first} Operator.
+
+[id="oadp-operator-fails-silently_{context}"]
+== OADP Operator fails silently
+
+The S3 buckets of an OADP Operator might be empty, but when you run the command `oc get po -n <OADP_Operator_namesapce>`, you see that Operator has a status of `Running.`  In such a case, the Operator is said to have _failed silently_ because it incorrectly reports that it is running.
```

Comment:
```suggestion
The S3 buckets of an OADP Operator might be empty, but when you run the command `oc get po -n <OADP_Operator_namesapce>`, you see that the Operator has a status of `Running.`  In such a case, the Operator is said to have _failed silently_ because it incorrectly reports that it is running.
```

Missing article - you might be missing the article “the” here?

Reply:
Good catches.